src/gf-platformid

```diff
@@ -47,15 +47,20 @@ coreos_gf download "${blscfg_path}" "${tmpd}"/bls.conf
 # Remove any platformid currently there
 sed -i -e 's, ignition.platform.id=[a-zA-Z0-9]*,,g' "${tmpd}"/bls.conf
 sed -i -e 's,^\(options .*\),\1 ignition.platform.id='"${platformid}"',' "${tmpd}"/bls.conf
-coreos_gf upload "${tmpd}"/bls.conf "${blscfg_path}"
 
 if [ "$basearch" = "s390x" ] ; then
+    # Before we re-run zipl make sure we have the firstboot options
+    sed -i -e 's,^\(options .*\),\1 ignition.firstboot rd.neednet=1 ip=dhcp\,dhcp6,' "${tmpd}"/bls.conf
```

Comment:
Instead of `sed`ing it in then out, I think I would feel better if we instead kept the `coreos_gf upload` higher up, then just upload our modified version as e.g. `bls.conf.tmp` and passed `--parmfile bls.conf.tmp` then deleted it.

Reply:
that is what i looked into doing initially, but the `--parmfile` requires all the other options like kernel and initrd which need to be parsed from the existing bls which seemed to be pretty tedious too..i will look into it again

Reply:
Ahh gotcha. Alternatively, you can rename `bls.conf` to `bls.conf.orig`, then upload the temporary one, then rename/overwrite `bls.conf.orig` to `bls.conf`.

Reply:
ok..was not aware that there was a renamefor guesfish but looks like there is! thanks ..will do that